src/mono/sample/wasm/browser-legacy/Wasm.Browser.LegacySample.csproj

```diff
@@ -1,12 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration>Debug</Configuration>
     <WasmCopyAppZipToHelixTestDir Condition="'$(ArchiveTests)' == 'true'">true</WasmCopyAppZipToHelixTestDir>
     <WasmMainJSPath>main.js</WasmMainJSPath>
     <DebugSymbols>true</DebugSymbols>
     <DebugType>embedded</DebugType>
     <WasmDebugLevel>1</WasmDebugLevel>
-    <WasmBuildNative>true</WasmBuildNative>
```

Comment:
Why is `WasmBuildNative` removed?

Reply:
Currently it's not necessary as we are still CJS. After the switch, the `<WasmEnableES6>false</WasmEnableES6>` will force it anyway.